src/mmw/js/src/draw/templates/aoiUpload.html

```diff
@@ -1 +1,21 @@
-<span id="dropbox">Drop a .shp file here.</span>
+<ul class="draw-pane-list">
+  <li class="draw-pane-list-item" role="presentation">
+    <div>
+        <span class="draw-button-title pull-left"
+              id="dropbox">
+            Drag and drop a file
+            <ul class="draw-pane-description-list">
+                <li>Must be a shapefile or geojson</li>
+                <li>Less than 200 MB</li>
+                <li>Only the first feature is used.</li>
+            </ul>
+        <button class="btn btn-success pull-left"
+                onclick="document.getElementById('draw-tool-file-upload-input').click();">
```

Comment:
Having it in the template was somewhat nice because it is explicit about the button being a dummy for the hidden input. Writing the `onclick` code as a string is not ideal though